handlers/test/unit/handlers/mock_yum.py

```diff
@@ -117,26 +124,43 @@ def __init__(self, *args, **kwargs):
         self.tsInfo = []
         self.repos = mock.Mock()
 
+    def _tx_member_for_instup(self, pkg, update=False):
+        # When installing or updating a package, packages in yum's "installonlypkgs" list
+        # have a special transaction state, as these are the packages that are never updated.
+        # Instead, they have multiple versions installed (e.g. kernel packages). This encapsulates
+        # the logic for the yum test mock: If one of the installonlypkgs is being installed or
+        # updated, use the special transaction state represented by TS_TRUEINSTALL. Otherwise,
+        # return the TS_INSTALL state if installing (update kwarg is False), or TS_UPDATE if
+        # updating (update kwarg is True).
+        if pkg.name in YumConf.installonlypkgs.default:
```

Comment:
Good thinking. this is only for the test mock of yum, so I think it would be better to just manually define the list in our tests and mention where it came from in a comment. I don't have a good reason to add this dependency here.